src/status_im/ui/screens/profile/views.cljs

```diff
@@ -29,9 +30,11 @@
   [toolbar/toolbar {}
    nil
    [toolbar/content-title ""]
-   [toolbar/actions
-    [(actions/opts [{:value #(dispatch [:my-profile/edit-profile])
-                     :text  (label :t/edit)}])]]])
+   [react/touchable-highlight
+    {:on-press #(dispatch [:my-profile/edit-profile])}
+    [react/view
+     [react/text {:style      styles/toolbar-edit-text
+                  :uppercase? component.styles/uppercase?} (label :t/edit)]]]])
```

Comment:
Would it makes sense to have a dedicated component for those uppercase text?

Reply:
We wouldn't gain much as we'd still need the style property for things like text color or font size, and possibly other props. So, we'd only trade the `:uppercase?` property for a different component name. E.g. the above would be: 

```clojure
[react/case-text {:style      styles/toolbar-edit-text} (label :t/edit)]
```

Also, it kind of seems wrong to have a different component for styling. All caps text on android isn't semantically different, it's just formatted differently, right?

Reply:
Not all text are uppercase? So there might be some subset of texts that have a semantic (title, header, ..). But that still to define with design team.

Reply:
That's possible. I haven't been able to catch the semantics, but that doesn't mean it's not there. Let's check with them so